cmd/podman/containers/exec.go

```
@@ -84,6 +87,12 @@ func execFlags(cmd *cobra.Command) {
 	flags.StringVarP(&execOpts.WorkDir, workdirFlagName, "w", "", "Working directory inside the container")
 	_ = cmd.RegisterFlagCompletionFunc(workdirFlagName, completion.AutocompleteDefault)
 
+	waitFlagName := "wait"
+	flags.StringVarP(&execOpts.Wait, waitFlagName, "", "0", "")
+	if err := flags.MarkHidden("wait"); err != nil {
```

You can ignore the error return in MarkHidden as we did below.